Formula/geocouch.rb

```diff
@@ -137,4 +139,6 @@ def caveats; <<-EOS.undent
     run 'brew info geocouch'.)
     EOS
   end
+
+  test {}
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!